edit/networking.k8s.io.ingress/Rules.vue

```diff
@@ -0,0 +1,147 @@
+<script>
+import { SERVICE } from '@/config/types';
+import SortableTable from '@/components/SortableTable';
+import { _VIEW } from '@/config/query-params';
+import Rule from './Rule';
+
+export default {
+  components: { Rule, SortableTable },
+
+  props: {
+    value: {
+      type:    Object,
+      default: () => {}
+    },
+
+    mode: {
+      type:    String,
+      default: 'edit'
+    },
+
+    serviceTargets: {
+      type:    Array,
+      default: () => []
+    }
+  },
+
+  computed: {
+    isView() {
+      return this.mode === _VIEW;
+    },
+    ruleHeaders() {
+      return [
+        {
+          name:      'path',
+          label:     this.t('ingress.rules.headers.path'),
+          formatter: 'Link',
+          value:     'pathLink'
+        },
+        {
+          name:      'target',
+          label:     this.t('ingress.rules.headers.target'),
+          formatter: 'Link',
+          value:     'targetLink',
+        },
+        {
+          name:  'port',
+          label: this.t('ingress.rules.headers.port'),
+          value: 'backend.servicePort'
+        }
+      ];
+    },
+    ruleRows() {
+      return this.value.spec.rules
+        .filter(rule => rule?.http?.paths)
+        .map((rule) => {
+          return {
+            ...rule,
+            http: { paths: this.withUrl(rule.host, rule.http.paths) }
+          };
+        });
+    },
+  },
+
+  methods: {
+    addRule() {
+      this.value.spec.rules = [...this.value.spec.rules, {}];
+      this.$forceUpdate();
+    },
+    removeRule(idx) {
+      const neu = [...this.value.spec.rules];
+
+      neu.splice(idx, 1);
+
+      this.$set(this.value.spec, 'rules', neu);
+      this.$forceUpdate();
+    },
+
+    updateRule(neu, idx) {
+      this.$set(this.value.spec.rules, idx, neu);
+    },
+
+    withUrl(host, paths = []) {
+      const rows = paths.map((path) => {
+        const serviceName = path?.backend?.serviceName;
+        const name = 'c-cluster-resource-namespace-id';
+        const params = {
+          resource:  SERVICE,
+          id:        serviceName,
+          namespace: this.value?.metadata?.namespace
+        };
+
+        const targetUrl = { name, params };
+        const pathUrl = `https://${ host }${ path?.path }`;
+
+        path.targetLink = {
+          url:     targetUrl,
+          text:    path?.backend?.serviceName
+        };
+        path.pathLink = {
+          url:     pathUrl,
+          text:    path?.path,
+          options: {
+            rel:    'nofollow noopener noreferrer',
+            target: '_blank'
+          }
```

Comment:
Why does `pathLink` have options and `targetLink` not?  

The `Link` formatter should just default to these options so that you have to explicitly disable them to if you really want an inline `target`-ed link (useful sometimes) or plain `rel` (likely never needed).

Reply:
One's a link to a resource so it doesn't leave dashboard. The other is a link to a host/path external to dashboard.

I'll go ahead and make that the default.